session.go

```diff
@@ -190,30 +209,50 @@ func (s *session) IsClosed() bool {
 }
 
 // set maximum pacakge length of every pacakge in (EventListener)OnMessage(@pkgs)
-func (s *session) SetMaxMsgLen(length int) { s.maxMsgLen = int32(length) }
+func (s *session) SetMaxMsgLen(length int) {
+	s.lock.Lock()
+	defer s.lock.Unlock()
```

Comment:
why not unlock after variable setting, the performance of which will be better

Reply:
I know it may be of better performance. However, this func is not invoked so frequently.